Assets/_Project/Code/Scripts/QuestionStage/AnswerControl.cs

```diff
@@ -0,0 +1,238 @@
+﻿using System;
+using System.Collections;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+using Unity.Netcode;
+using System.Linq;
+public class AnswerControl : NetworkBehaviour
```

Comment:
to są kontrollery więc powinno być AnswerController, zmień nazwę pliku adekwatnie do nazwy klasy